### PR TITLE
Data file text updates

### PR DESCRIPTION
**Pull request by jongio**
First update to get consistency with Display and Service names in same files and across files.

### CODE REVIEW

**Comment by jongio** [comment]
I'm converting to draft because I want to get more consistency in Display Name and Service Name across old and new.

**Review by czubair — APPROVED** [review]

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -1,217 +1,217 @@
...
+"azure-mgmt-healthcareapis","0.1.0","","azure-mgmt-healthcareapis","Health Care APIs","healthcareapis","NA","","","",""
+"azure-mgmt-hybridcompute","2.0.0","","azure-mgmt-hybridcompute","Hybrid Compute","hybridcompute","NA","","","",""
+"azure-mgmt-hybridkubernetes","0.1.0","","azure-mgmt-hybridkubernetes","Hybrid Kubernetes","hybridkubernetes","NA","","","",""
+"azure-mgmt-imagebuilder","0.4.0","","azure-mgmt-imagebuilder","Compute","compute","NA","","","",""
+"azure-mgmt-iotcentral","3.1.0","","azure-mgmt-iotcentral","IoT Hub","iothub","","","","",""
+"azure-mgmt-iothub","0.12.0","","azure-mgmt-iothub","IoT Hub","iothub","https://docs.microsoft.com/python/api/overview/azure/iot","","","",""
+"azure-mgmt-iothubprovisioningservices","0.2.0","","azure-mgmt-iothubprovisioningservices","IoT Hub","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-iothubprovisioningservices_0.2.0/azure-mgmt-iothubprovisioningservices/","NA","","","",""
+"azure-mgmt-kubernetesconfiguration","0.2.0","","azure-mgmt-kubernetesconfiguration","Kubernetes Configuration","kubernetesconfiguration","NA","","","",""
+"azure-mgmt-kusto","0.10.0","","azure-mgmt-kusto","Kusto","kusto","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:141]:
```suggestion
"azure-mgmt-kusto","0.10.0","","azure-mgmt-kusto","Kusto","kusto","","NA","","",""
```

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -50,23 +50,23 @@
...
-"@azure/arm-appconfiguration","1.3.0","","@azure/arm-appconfiguration","Appconfiguration","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/arm-appconfiguration","NA","NA","","",""
-"@azure/arm-appinsights","3.0.0","","@azure/arm-appinsights","Applicationinsights","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/applicationinsights/arm-appinsights","NA","NA","","",""
+"@azure/arm-apimanagement","6.0.0","","@azure/arm-apimanagement","API management","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/apimanagement/arm-apimanagement","NA","NA","","",""
+"@azure/arm-appconfiguration","1.3.0","","@azure/arm-appconfiguration","App Configuration","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/arm-appconfiguration","NA","NA","","",""
+"@azure/arm-appinsights","3.0.0","","@azure/arm-appinsights","Application Insights","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/applicationinsights/arm-appinsights","NA","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:55]:
```suggestion
"@azure/arm-appinsights","3.0.0","","@azure/arm-appinsights","Application Insights","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/applicationinsights/arm-appinsights","NA","NA","","",""
```

File: _data/releases/latest/js-packages.csv [hunk 3]
```diff
@@ -46,7 +46,7 @@
 "@autorest/test","0.1.69","","Test","Test","https://github.com/Azure/autorest.test","NA","NA","","false","",""
 "@autorest/test-server","3.0.27","","Test - Server","Test ","NA","NA","NA","","false","",""
 "@azure/abort-controller","1.0.1","","Abort - Controller","Core","core","NA","","","false","",""
-"@azure/ai-anomaly-detector","","3.0.0-beta.2","Anomaly Detector","Anomaly Detector","NA","","","","false","","Will replace: @azure/cognitiveservices-anomalydetector"
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv]: we should probably also mark this as a client library as well.

**Comment by weshaggard** [comment]
FYI I had to fix some spring links in https://github.com/Azure/azure-sdk/pull/1979 so you will need to pull those changes directly.

**Comment by jongio** [comment]
@weshaggard - Could you do a PR to this PR that includes the changes to take the new "New" field into account?  

Client = "New = true, Type = client"
Mgmt = "New = true, Type = mgmt"
All = "No New or Type filter"

**Review by weshaggard — APPROVED** [review]

**Pull request merged**